Intro example at /hello (#62)
cognitedata/javascript-getting-started: hello-gearbox/README.md

# Hello Gearbox

> > Introductory example app demonstrating how to access the `Cognite Data Fusion Platform` with the `JavaScript SDK` and the `Cognite Gearbox` component library.

Reviewer: links to all of them would be appreciated :)

Reviewer: you have them underneath, so just move them here (usually we put the link first time we mention something, after that it's not needed anymore)

> > Introductory example app demonstrating how to access the `Cognite Data Fusion Platform` with the `JavaScript SDK` and the `Cognite Gearbox` component library.

Reviewer: ```suggestion
Introductory example app demonstrating how to access the `Cognite Data Fusion Platform` with the `JavaScript SDK` and the `Cognite Gearbox` component library.
```

Author: Fixed

> Run with:

Reviewer: ```suggestion
Install npm modules:

`yarn` or `npm install`

Start application:

`yarn start`or `npm start`
```

```shell
npm install

> HTTPS=true npm start

Reviewer: and then this can be removed

# Hello Gearbox

> Introductory example app demonstrating how to access the Cognite [Data Fusion Platform](https://docs.cognite.com/) with the [JavaScript SDK](https://www.npmjs.com/package/@cognite/sdk) and the [Gearbox](https://www.npmjs.com/package/@cognite/gearbox) component library.

Reviewer: ```suggestion
Introductory example app demonstrating how to access the [Cognite Data Fusion](https://docs.cognite.com/) with the [JavaScript SDK](https://www.npmjs.com/package/@cognite/sdk) and the [Gearbox](https://www.npmjs.com/package/@cognite/gearbox) component library.
```

> Introductory example app demonstrating how to access the Cognite [Data Fusion Platform](https://cognite.com/products/cognite-data-fusion) with the [JavaScript SDK](https://github.com/cognitedata/cognitesdk-js) and the [Gearbox](https://github.com/cognitedata/gearbox.js) component library.

Reviewer: ```suggestion
Introductory example app demonstrating how to access the Cognite [Data Fusion Platform](https://docs.cognite.com/) with the [JavaScript SDK](https://www.npmjs.com/package/@cognite/sdk) and the [Gearbox](https://www.npmjs.com/package/@cognite/gearbox) component library.
```
